### DISCUSSION TITLE
Welcome to lcobucci/jwt Discussions!

### DISCUSSION THREAD

**Discussion by lcobucci** [lcobucci/jwt#603] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jgivoni** [comment]
Hello there, I'm Jakob and I just found this page because I'm trying to upgrade lcobucci/jwt to v4! I'm hoping to start a conversation about an issue I'm having.

**Reply by lcobucci** [reply]
@jgivoni welcome! :wave:

**Comment by andrewsauder** [comment]
Does anyone have an example of using validation constraints? The documentation states clearly "You MUST provide at least one constraint, otherwise `\Lcobucci\JWT\Validation\NoConstraintsGiven` exception will be thrown," but seems to miss giving an example of how to use the various bundled constraints in `\Lcobucci\JWT\Validation\Constraint\`
